src/components/menus/Items.ts

```diff
@@ -133,6 +141,24 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
                 text: "TOSS"
             }
         ];
+        if (this.gameStarter.features.heldItems) {
+            options.push({
+                callback: (): void => {
+                    const category = this.gameStarter.itemsHolder.getItem(listing.item).category;
+                    if (category === "HM") {
+                        return;
+                    }
+                    if (category === "TM") {
```

Comment:
*(so this comment doesn't get lost)* - remove the `return;`s for HM/TM categories.